apps/client/package.json

```diff
@@ -21,6 +22,7 @@
     "aws-amplify": "^5.0.12",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-intl": "^6.3.0",
```

Comment:
Can `react-intl` work with dependencies, e.g. AppKit?

Reply:
It will need to be setup, but definitely. App Kit is being transitioned to 100% React.

Also, `react-intl` is simply a wrapper around `formatjs`, which can be run on a node process using the same generated strings. In fact, I think we will want to integrate `formatjs` into `core` to internationalize our response messages. Doing that, we could use messages directly sent by the back-end in the client-side error UX, instead of adding additional mapping to messages on the client for various status codes. At the moment, I am not aware of how to do such.